app/templates/content/index.hbs

```diff
@@ -90,22 +90,28 @@
                         </button>
                     </div>
 
-                    {{#if model.links.preprint_doi}}
-                        <div class="p-t-xs">
-                            <h4 class="p-v-md f-w-md"><strong>{{t "content.preprint_doi"}}</strong></h4>
+                    <div class="p-t-xs">
+                        <h4 class="p-v-md f-w-md"><strong>{{t "content.preprint_doi"}}</strong></h4>
+                        {{#if model.preprintDoiUrl}}
                             {{#if model.preprintDoiCreated}}
-                                <a href={{model.links.preprint_doi}} onclick={{action "click" "link" "Content - Preprint DOI" model.links.preprint_doi}}>{{extract-doi model.links.preprint_doi}}</a>
+                                <a href={{model.preprintDoiUrl}} onclick={{action "click" "link" "Content - Preprint DOI" model.preprintDoiUrl}}>{{extract-doi model.preprintDoiUrl}}</a>
                             {{else}}
-                                <p> {{extract-doi model.links.preprint_doi}} </p>
+                                <p> {{extract-doi model.preprintDoiUrl}} </p>
                                 <p> {{t "content.preprint_pending_doi_minted"}} </p>
                             {{/if}}
-                        </div>
-                    {{/if}}
+                        {{else}}
+                            {{#if (not node.public)}}
+                                {{t 'content.preprint_pending_doi'}}
+                            {{else if (and model.provider.reviewsWorkflow (not model.isPublished))}}
+                                {{t 'content.preprint_pending_doi_moderation'}}
+                            {{/if}}
+                        {{/if}}
+                    </div>
 
-                    {{#if model.doi}}
+                    {{#if model.articleDoiUrl}}
                         <div class="p-t-xs">
                             <h4 class="p-v-md f-w-md"><strong>{{t "content.article_doi"}}</strong></h4>
-                            <a href={{doiUrl}} onlick={{action "click" "link" "Content - DOI" doiUrl}}>{{model.doi}}</a>
+                            <a href={{model.articleDoiUrl}} onlick={{action "click" "link" "Content - DOI" model.articleDoiUrl}}>{{model.articleDoiUrl}}</a>
```

Comment:
The text here should maybe be `{{extract-doi model.articleDoiUrl}}`, to be consistent with the previous section and what was displayed before.

Reply:
This is not a bad idea, but also note that the `extractDoiFromString` utility (which the extract-doi helper wraps) is [used to sanitize user input](https://github.com/CenterForOpenScience/ember-osf-preprints/blob/develop/app/controllers/submit.js#L792). It's always good to make sure, though, especially since the API doesn't enforce this and we may have bulk uploads in the near future. We may want to rename `model.articleDoiUrl` because it is actually just a bare DOI and not a full URL.

Reply:
Also, I just noticed that you are switching the link from `doiUrl` to `model.articleDoiUrl`. This will result in a bad link. `doiUrl` is a computed that [makes the bare DOI into a URL](https://github.com/CenterForOpenScience/ember-osf-preprints/blob/develop/app/controllers/content/index.js#L186).

Reply:
Looking at [the API](https://api.osf.io/v2/preprints/), `model.articleDoiUrl` (aka `model.links.doi`) is the full URL, while `model.doi` is just the bare DOI.

Reply:
Wait, now I'm confused, we a doi attribute AND a doi link?

Reply:
Yep! Good times. That confusion is why I'd rather not use either, and stick to `articleDoiUrl`

Reply:
If the API is giving us the full url, we should definitely use that, but we should also then remove the `doiUrl` computed.

Reply:
[Totally agree](https://github.com/CenterForOpenScience/ember-osf-preprints/pull/502/files#diff-1940ae5c2ed759c5fcdbeea83483b130L185)

Reply:
Nevermind, ignore everything I've said here. :)

Reply:
Only question is whether to use `{{extract-doi model.articleDoiUrl}}` or `{{model.doi}}` for the text since they should be equivalent.

Reply:
Yeah, either's valid. But because it's so confusingly named, I lean away from using `model.doi` at all ever.

I wouldn't be opposed to renaming the ember model field to `articleDoi`, though...